tests/gnrc_sock_dns/README.md

```diff
@@ -0,0 +1,25 @@
+# Overview
+
+This folder contains a test application for RIOT's sock-based DNS client.
+
+# How to test with native
+
+Setup up a tap interface:
+
+    # sudo ip tuntap add dev riot0 mode tap user $(id -u -n)
```

Comment:
Here and below: `#` usually denotes that a root shell is required. `sudo` gives you root and the compiled application shouldn't require root, so I would prefer `$`.